### PR TITLE
Enabling AVR support for certain ATTiny devices

### PR DESCRIPTION
**Pull request by eadgbear**
Small changes to get MySensors working on the Digispark Pro (based on the attiny167).

Any feedback would be greatly appreciated!

### CODE REVIEW

File: hal/architecture/AVR/MyHwAVR.h [hunk 1]
```diff
@@ -28,7 +28,12 @@
 #include <avr/wdt.h>
 #include <avr/boot.h>
 #include <util/atomic.h>
-#include <SPI.h>
+
+#if defined( __AVR_ATtiny25__ ) || defined( __AVR_ATtiny45__ ) || defined( __AVR_ATtiny85__ ) || defined( __AVR_ATtiny87__ ) || defined( __AVR_ATtiny167__ )	
+    #include <TinySoftwareSPI.h>
```
Comment (mfalkvidd) [inline hal/architecture/AVR/MyHwAVR.h:33]: What library is this? Is it part of the standard Arduino library or must it be installed manually?

Comment (eadgbear) [reply hal/architecture/AVR/MyHwAVR.h:33]: This library comes along with most (can't guarantee all) Tiny implementations that I've tried. Makes MySensors work for my Digispark Pro (ATTinty167) like a charm.

Comment (garudaonekh) [reply hal/architecture/AVR/MyHwAVR.h:33]: Hello, TinySoftwareSPI.h looks like deprecated. Can be replaced with other spi library?

**Comment by garudaonekh** [comment]
I can't get it to work with my Attiny167. Get many errors in MyMainAVR.cpp

**Comment by eadgbear** [comment]
> I can't get it to work with my Attiny167. Get many errors in MyMainAVR.cpp

Can you share what some of them are?

**Review by garudaonekh — COMMENTED** [review]
> > I can't get it to work with my Attiny167. Get many errors in MyMainAVR.cpp
> 
> Can you share what some of them are?

`In function 'void hwPowerDown(uint8_t)':
../MySensors/hal/architecture/AVR/MyHwAVR.cpp:93:26: error: 'WDTCSR' was not declared in this scope
  const uint8_t WDTsave = WDTCSR;
                          ^~~~~~

/../MySensors/hal/architecture/AVR/MyHwAVR.cpp: In function 'uint16_t hwCPUFrequency()':
/../MySensors/hal/architecture/AVR/MyHwAVR.cpp:305:26: error: 'WDTCSR' was not declared in this scope
  const uint8_t WDTsave = WDTCSR;
                          ^~~~~~
..MySensors/hal/architecture/AVR/MyMainAVR.cpp: In function 'int main()':
/Users/chamroeun/Documents/Arduino/libraries/MySensors/hal/architecture/AVR/MyMainAVR.cpp:34:7: error: 'serialEventRun' was not declared in this scope
   if (serialEventRun) {`


Please note that I use AttinyCore

File: hal/architecture/AVR/MyHwAVR.h (hunk 1 above)
Comment (garudaonekh) [reply hal/architecture/AVR/MyHwAVR.h:33]: TinySoftwareSPI.h was removed from AttinyCore. And I don't know the reason

**Pull request closed without merging**